modin/numpy/utils.py

```diff
@@ -0,0 +1,30 @@
+# Licensed to Modin Development Team under one or more contributor license agreements.
+# See the NOTICE file distributed with this work for additional information regarding
+# copyright ownership.  The Modin Development Team licenses this file to you under the
+# Apache License, Version 2.0 (the "License"); you may not use this file except in
+# compliance with the License.  You may obtain a copy of the License at
+#
+#     http://www.apache.org/licenses/LICENSE-2.0
+#
+# Unless required by applicable law or agreed to in writing, software distributed under
+# the License is distributed on an "AS IS" BASIS, WITHOUT WARRANTIES OR CONDITIONS OF
+# ANY KIND, either express or implied. See the License for the specific language
+# governing permissions and limitations under the License.
+
+"""Collection of array utility functions for internal use."""
+
+import modin.pandas as pd
+import modin.numpy as np
```

Comment:
You are importing a whole `np` module, which itself imports `utils.py` this can very easily spawn cyclic imports.
```suggestion
from modin.numpy.arr import array
```

Reply:
Makes sense to me!

Reply:
Tried this and got a circular import error. Reverted!

Reply:
oops :)